core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -1192,9 +1192,7 @@ class SQLPlanParserSuite extends BaseTestSuite {
     }
   }
 
-  test("ParseUrl is supported except that for parse_url_query") {
-    // parse_url(*,QUERY,*) should cause the project to be unsupported
-    // the expression will appear in the unsupportedExpression summary
+  test("ParseUrl is supported") {
```

Comment:
Does this mean we support `parse_url` for all values of `partToExtract`?. 

In the supportedExpr.csv we say it is partially supported:

https://github.com/NVIDIA/spark-rapids-tools/blob/c58085136e160cd164013bd263be5a74a3e7999a/core/src/main/resources/supportedExprs.csv#L410

Reply:
yeah, this is the same in the plugin csv files. I believe it means that it is limited support because
`REF | FILE | AUTHORITY | USERINFO`  parts are not supported + there is some different results between CPU/GPU in query parts.

Q tool used to disqualify query expression which is incorrect. 

I can follow up with another commit to detect unsupported parts or we can just simplify our handling to `parse_url` considering it supported which won't be too bad for accuracy. I am more inclined to simplify it.
https://github.com/NVIDIA/spark-rapids/pull/10211/files
https://github.com/NVIDIA/spark-rapids/pull/10740/files

Reply:
I think the simplification makes sense. Thanks @amahussein

Reply:
My conscience hurt. I updated the PR description and implemented the complete fix.
Thanks @parthosa